src/Jobs/UpdatePriceJob.php

```diff
@@ -46,9 +47,20 @@ public function handle(ChecksMagentoExistence $checksMagentoExistence): void
 
         $data = $model->getData();
 
-        $this->handleBasePrices($data);
-        $this->handleTierPrices($data);
-        $this->handleSpecialPrices($data);
+        Bus::batch(array_filter([
+            $this->handleBasePrices($data),
+            $this->handleTierPrices($data),
+            $this->handleSpecialPrices($data),
+        ]))
+            ->name('update-price-jobs')
```

Comment:
Could you use a human readable name that includes the SKU? There will be a batch per product.
Something like "Magento Price Update {sku}"